styles.css

```diff
@@ -0,0 +1,40 @@
+/* .main{
+    text-align: center;
+}
+.img-div{
+    width: 60px;
+    height: 60px;
+    position: relative;
+    padding-right: 30px;
+}
+.tribute-info{
+    text-align: center;
+    border: 2px orangered;
+} */
+
+*{
+    box-sizing: 0;
```

Comment:
Please zero(0) is not a valid value for the box-sizing property.

Reply:
Good job by telling the student **WHAT** he should change but also explain to him **WHY** he needs to change that thing. It might be obvious to you but there's a chance that others won't be thinking in that way. That's why it's better to also explain the reason why he should make the changes. 😄 👍🏻

You can also suggest to him the valid properties for box-sizing or add the link to an article that explains about the box-sizing property. 👍🏻

Reply:
Good job by telling the student WHAT he should change but also explain to him WHY he needs to change that thing. It might be obvious to you but there's a chance that others won't be thinking in that way. That's why it's better to also explain the reason why he should make the changes. 😄 👍🏻

You can also suggest to him the valid properties for box-sizing or add the link to an article that explains about the box-sizing property. 👍🏻